### PR TITLE
JBDS-4439 Detect free space available on disk for macOS

### PR DESCRIPTION
**Pull request by sudhirverma**

### CODE REVIEW

File: browser/services/platform.js [hunk 1]
```diff
@@ -133,6 +133,34 @@ class Platform {
...
+        let disk = path.parse(location).root.charAt(0);
+        return pify(child_process.exec)(`powershell -command "& {(Get-WMIObject Win32_Logicaldisk -filter \"deviceid=\`'${disk}:\`'\") }"`).then((stdout) => {
+          return Promise.resolve(Number.parseInt(stdout));
+        }).catch(()=>{
+          return Promise.resolve();
+        });
+      },
+      darwin: ()=> {
+        return pify(child_process.exec)(`df -k ${location}`).then((stdout) => {
+          var lines = stdout.split('\n');
```
Comment (dgolovin) [inline browser/services/platform.js]: I don't think we need cycle here, just 
``` JavaScript
let split = lines[1].replace( / +/g, ' ' ).split(' ');
// split[0] -> disk
// split[1] - > total 1k blocks
// split[2] - > total 1k blocks used
// split[3] - > total 1k blocks available
return Promise.resolve(Number.parseInt(split[3]));
```
we just need return the same units fro macOS and Windows.

**Comment by sudhirverma** [comment]
test this

**Comment by sudhirverma** [comment]
test this

**Comment by dgolovin** [comment]
test this

**Pull request merged**